Content.Server/Xenoarchaeology/Equipment/Systems/ArtifactCrusherSystem.cs

```diff
@@ -61,11 +64,14 @@ private void OnPowerChanged(Entity<ArtifactCrusherComponent> ent, ref PowerChang
 
     public void StartCrushing(Entity<ArtifactCrusherComponent, EntityStorageComponent> ent)
     {
-        var (_, crusher, _) = ent;
+        var (uid, crusher, _) = ent;
 
         if (crusher.Crushing)
             return;
 
+        if (crusher.AutoLock)
+            _popup.PopupEntity(Loc.GetString("artifact-crusher-autolocks-enable"), uid);
+
```

Comment:
same deal here. As a note, having a popup play every time seems really annoying so maybe just have it on the attempt open event. That way the person inside can see that it's locked for all glorious 10 seconds they're getting crushed.